pysatMissionPlanning/tests/test_core.py

```diff
@@ -15,6 +15,6 @@ def teardown(self):
     def test_module_loading(self):
         """Check if instrument module can be loaded as a pysat instrument"""
 
-        testInst = pysat.Instrument(module=self.module)
+        testInst = pysat.Instrument(inst_module=self.module)
```

Comment:
You could also test that testInst.platform and testInst.name match the corresponding attributes of self.module . I think that would've at least caught this issue.

Could also test that the correct load, clean, etc. from self.module are also present as testInst._list_rtn etc

Reply:
I noticed that the coverage went up a bit after fixing this.

Reply:
Cool. I like my test suggestions, but those are probably better within pysat, and perhaps out of scope.

Reply:
Ok. Will think about bout this as part of the next one.